taskApp/src/app/pages/list-tasks/list-tasks.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+
+@Component({
+  selector: 'app-list-tasks',
+  templateUrl: './list-tasks.component.html'
+})
+
+export class ListTasksComponent{
+
+  /* An empty array that is responsible 
+  to add a division */
+  public items = []; 
+
+  public newTask; 
+
+
+  /* When input is empty, it will 
+    not create a new division */
+  public addToList() { 
+      if (this.newTask == '') { 
+      } 
+      else { 
+          this.items.push(this.newTask); 
```

Comment:
Since we have the ability in your API to save tasks how about you get this connected to your API now.

Reply:
In case you need some docs on how to do this: https://angular.io/guide/http

Reply:
So this is messy. For add task and delete task I received a unhandled promise warning. The operations occur, you just have to refresh the web browser. Also I'm passing an object, I can't figure out how to just have the 'text' attribute show

Reply:
This sounds like on result from the add/delete you need to run getting all tasks again in the .then for the promise.  I'm unsure what you mean by an object.  Could you explain more?  I'm up for a call if it's better to say/show.

Reply:
Are you referring to the object in the html?